Post presentation cleanup

Michael-Kochis opened the pull request.

max-nguyen-revature reviewed (commented):
these also still have some setters that shouldnt be in the user model i think, right?

Michael-Kochis commented:
Thought I fixed that, moved everything to setEmail and getEmail.

On Tue, Nov 2, 2021 at 4:07 PM max-nguyen-revature ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
>
> these also still have some setters that shouldnt be in the user model i
> think, right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Revature-Reverb/backend/pull/13#pullrequestreview-795916758>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOG23MBTJAUGQ3JYHY57FXTUKBOJ5ANCNFSM5HHLNGBQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

alexjohnson9551 commented:
The backend user model no longer has a firstname or other profile information (and does not even reference the profile as an object. Rather the profile object and associated table will have a foreign key to a user id.) So I'm going to close this pull request, it would break the user controller and these were just for testing anyway.
